unit_tests/storage/test_storage_facility.cpp

```diff
@@ -56,20 +56,25 @@ struct static_type_tests {
 template <>
 struct static_type_tests<backend::cuda> {
     using storage_traits_t = storage_traits<backend::cuda>;
+#ifdef __HIPCC__
+    using alignment_t = alignment<16>;
+#else
+    using alignment_t = alignment<32>;
+#endif
 
     /*########## STORAGE INFO CHECKS ########## */
     // storage info check
     typedef storage_traits_t::storage_info_t<0, 3, halo<1, 2, 3>> storage_info_ty;
     GT_STATIC_ASSERT(
         (is_storage_info<storage_info_ty>::type::value), "is_storage_info metafunction is not working anymore");
-    GT_STATIC_ASSERT((std::is_same<storage_info_ty,
-                         storage_info<0, layout_map<2, 1, 0>, halo<1, 2, 3>, alignment<32>>>::type::value),
+    GT_STATIC_ASSERT(
+        (std::is_same<storage_info_ty, storage_info<0, layout_map<2, 1, 0>, halo<1, 2, 3>, alignment_t>>::type::value),
```

Comment:
I don't think this test is very useful anyway, but maybe we just shouldn't check the alignment (or query the alignment from the storage info as proposed above).

Reply:
But it looks likes it’s the exact purpose of this test to check those types? So either it should include alignment or otherwise we can drop the test, right?

Reply:
I don't think the test makes much sense.

Reply:
I removed it…